api/src/main/java/org/openmrs/module/patientflags/db/FlagDAO.java

```diff
@@ -50,6 +50,22 @@
 	 * @throws DAOException
 	 */
 	public Flag getFlag(Integer flagId) throws DAOException;

```

Comment:
You need to have at least a line between methods.

Reply:
Done